src/ansys/hps/client/rms/models.py

```diff
@@ -34,683 +34,836 @@
+        description='Get project assignments from the server instead of using the locally set values',
+        title='Project Server Select',
     )
     project_list: Optional[List[str]] = Field(
         [],
-        description="IDs of projects that the evaluator should work on, in order",
-        title="Project List",
+        description='IDs of projects that the evaluator should work on, in order',
+        title='Project List',
     )
     project_assignment_mode: Optional[str] = Field(
-        "all_active",
-        description="Specifies how the evaluator selects projects to work on.             One of: disabled, all_active, list",
-        title="Project Assignment Mode",
+        'all_active',
+        description='Specifies how the evaluator selects projects to work on.             One of: disabled, all_active, list',
+        title='Project Assignment Mode',
     )
     context: Optional[Context] = Field(
-        {"custom": {}}, description="Runtime properties to pass to executed tasks."
+        {'custom': {}, 'use_local_scratch': False},
+        description='Runtime properties to pass to executed tasks.',
     )
 
 
 class EvaluatorConfigurationUpdate(BaseModel):
-    id: Optional[str] = Field(None, description="Unique database ID (read-only).", title="Id")
+    id: Optional[str] = Field(None, description='Unique database ID (read-only).', title='Id')
     evaluator_id: Optional[str] = Field(
-        None, description="ID of the parent evaluator (read-only).", title="Evaluator Id"
+        None, description='ID of the parent evaluator (read-only).', title='Evaluator Id'
     )
     last_modified: Optional[datetime] = Field(
-        None, description="Last modified time", title="Last Modified"
+        None, description='Last modified time', title='Last Modified'
     )
-    working_directory: Optional[str] = Field(None, title="Working Directory")
+    working_directory: Optional[str] = Field(None, title='Working Directory')
     local_file_cache_max_size: Optional[int] = Field(
-        None, description="Maximum allowed cache size in bytes.", title="Local File Cache Max Size"
+        None, description='Maximum allowed cache size in bytes.', title='Local File Cache Max Size'
     )
-    max_num_parallel_tasks: Optional[int] = Field(None, title="Max Num Parallel Tasks")
+    max_num_parallel_tasks: Optional[int] = Field(None, title='Max Num Parallel Tasks')
     task_directory_cleanup: Optional[TaskDirectoryCleanupEnum] = Field(
-        None, title="Task Directory Cleanup"
+        None, title='Task Directory Cleanup'
     )
-    resources: Optional[EvaluatorResources] = {"custom": {}}
+    resources: Optional[EvaluatorResources] = {'custom': {}}
     name: Optional[str] = Field(
         None,
-        description="Update the name of the evaluator (updating the registration).",
-        title="Name",
+        description='Update the name of the evaluator (updating the registration).',
```

Comment:
Again--the description is normally a noun. Rather than starting with a verb, can we rewrite as: "Name to update the evaluator to. Chaning the name updates the registration."